emscripten.py

```diff
@@ -448,6 +452,15 @@ def get_metadata(infile, outfile, modify_wasm, args):
   return metadata
 
 
+def create_sourcemapping_url_file(infile, url):
+  'Create a file with the contents of the sourceMappingURL section'
```

Comment:
Use triple double quotes for doc strings.

Reply:
Done